scheds/rust/scx_layered/src/config.rs

```diff
@@ -113,6 +113,8 @@ pub struct LayerCommon {
     #[serde(default)]
     pub perf: u64,
     #[serde(default)]
+    pub idle_resume_us: u32,
```

Comment:
Can we do `Option<u32>`? We aren't consistent about it but let's use them for new ones at least.